static/bidder-params/criteo.json

```diff
@@ -23,6 +23,16 @@
             "type": "integer",
             "description": "Impression's network ID, preferred.",
             "minimum": 0
+        },
+        "pubid": {

```

Comment:
Please add test cases to your `params_test.go`.

Reply:
@vraybaud can you add the following invalid test cases as well?
- `{"zoneid": 0, "networkid": 0, "pubid": ""}`
- `{"zoneid": 0, "networkid": 0, "pubid": null}`
- `{"zoneid": 0, "networkid": 0, "uid": null}`

Reply:
Thanks, done !